pkg/storegateway/bucket.go

```diff
@@ -536,8 +537,8 @@ type seriesChunks struct {
 	chks []storepb.AggrChunk
 }
 
-// Series implements the storepb.StoreServer interface.
-func (s *BucketStore) Series(req *storepb.SeriesRequest, srv storepb.Store_SeriesServer) (err error) {
+// Series implements the storegatewaypb.StoreGatewayServer interface.
+func (s *BucketStore) Series(req *storepb.SeriesRequest, srv storegatewaypb.StoreGateway_SeriesServer) (err error) {
```

Comment:
Note to reviewers: this is **not** a breaking change because `storepb.Store_SeriesServer` and `storegatewaypb.StoreGateway_SeriesServer` interfaces are **identical**.